Model card issue#1125 (#1129)
huggingface/hub-docs: docs/hub/model-card-annotated.md

## Training Procedure [optional]

> _When you want to know what hardware you'll need to fine-tune a model, consider the following factors: the number of parameters in the model and the training regime you plan to use._

Reviewer: ```suggestion
_When you want to know what hardware you'll need to train or fine-tune a model, consider the following factors: the number of parameters in the model and the training regime you plan to use._
```

Maybe?

> _e.g A model with 3B parameters and fp32 precision format needs at least 48GB of GPU memory, while bf16 requires at least 24GB of memory with Amphere or higher hardware. Mixed pf16 requires at least 54GB of GPU memory._

Reviewer: ```suggestion
_e.g A model with 3B parameters and fp32 precision format needs at least 48GB of GPU memory, while bf16 requires at least 24GB of memory with Ampere or higher hardware. Mixed fp16 requires at least 54GB of GPU memory._
```

Reviewer: These numbers sound a bit high to me. In any case, they depend on a number of factors like optimizer choice. Should the recommended optimizer be a part of the training_regime data?

_Detail tokenization, resizing/rewriting (depending on the modality), etc._

### Training Hyperparameters

* **Training regime:** training_regime`
  

> _Detail the model training process, specifically the type of precision used - whether it is **fp32/fp16/bf16** - and whether it is **mixed or non-mixed precision**?_

Reviewer: ```suggestion
_Detail the model training process, specifically the type of precision used - whether it is **fp32/fp16/bf16** - and whether it is **mixed or non-mixed precision**_
```
